modules/standard/fun/ding_controller.py

```diff
@@ -0,0 +1,137 @@
+import random
+
+from core.command_param_types import Any
+from core.decorators import command, instance
+from core.registry import Registry
+
+
+@instance()
+class DingController:
+    """
+        Port of Budabot fun module
+    """
+
+    def inject(self, registry: Registry):
+        self.command_alias_service = registry.get_instance(
+            "command_alias_service")
+
+    def start(self):
+        self.command_alias_service.add_alias("levelup", "ding")
+
+    @command(command="ding", params=[Any("level", is_optional=True)], access_level="all",
```

Comment:
Can do `Int("level", is_optional=True)`, and it'll only accept numbers

Reply:
Yup I had that initially, but it wont resolve negative numbers? I didn't want to remove the sarcastic/witty comments that would be returned when someone enters a negative number.

Reply:
Aha. Should be a quick fix to the Int-decorator type. It's probably not taking the hyphen/minus sign into account in the regex.